### ISSUE TITLE
Systemic sclerosis

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#6089]
**Preferred term label:**
Systemic sclerosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is a disease rather than a phenotypic feature.
See this MONDO term: https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005100
